### DISCUSSION TITLE
Organization sponsorship

### DISCUSSION THREAD

**Discussion by ahmetarsiv** [community/community#42819] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello I hope you are well,

I am trying to activate the sponsor panel for my Github organization: I have completed the profile information and the stripe account without any problems, but it has not been activated for a long time. Can anyone help with this information? or does anyone know the Github support email account?

![sponsor](https://user-images.githubusercontent.com/101526596/209910922-92820906-6d2d-4d99-9422-a58ce9e6dd91.png)

I've been waiting like this for about a month.

**Comment by radar-blip** [comment]
You can contact support here: [Support Page](https://support.github.com/request)

Here is another link that may help you: https://support.github.com/?tags=dotcom-footer&q=sponsers
